valhalla/baldr/graphtile.h

```diff
@@ -563,16 +563,9 @@ class GraphTile {
     return de->speed();
   }
 
-  inline uint32_t GetCongestion(const DirectedEdge* de) const {
-    // TODO(danpat): this needs to consider the time - we should not use live speeds if
-    //               the request is not for "now", or we're some X % along the route
-    // TODO(danpat): for short-ish durations along the route, we should fade live
-    //               speeds into any historic/predictive/average value we'd normally use
+  inline const volatile traffic::Speed& GetLiveSpeed(const DirectedEdge* de) const {
```

Comment:
at the very least this function and the other one that uses the index `getTrafficForDirectedEdge` should have the same name. this follows the pattern that we use in the rest of the code. see functions like `directededge` `nodeinfo` etc. also i think maybe in that same style it shouldnt use the `getCamalCase` it should probably use all lowercase and the name of the class its getting.. although that is complicated in this case since its in its own namespace (another nit). it should probably be `baldr::TrafficSpeed` and then this function should probably be

    inline const volatile traffic::Speed& trafficspeed(const DirectedEdge* de) const {

And the other one should be:

    inline const volatile traffic::Speed& trafficspeed(uint32_t directed_edge_index) const {

Reply:
Adjusted.